COPYRIGHT.txt

```diff
@@ -307,11 +307,6 @@ Comment: Fast Filtering of Reflection Probes
 Copyright: 2016, Activision Publishing, Inc.
 License: Expat
 
-Files: ./thirdparty/misc/easing_equations.cpp
-Comment: Robert Penner's Easing Functions
-Copyright: 2001, Robert Penner
-License: BSD-3-clause
```

Comment:
This should be kept, but with the new path, and:
```
Copyright: 2001, Robert Penner
 2007-2021 Juan Linietsky, Ariel Manzur.
 2014-2021 Godot Engine contributors.
```

Reply:
And this should be moved to keep alphabetical order of `Files` of course, and change the `License` to `Expat`.